lib/plugins/place_block.js

```diff
@@ -73,6 +73,8 @@ function inject (bot, { version }) {
 
     if (oldBlock.type === newBlock.type) {
       throw new Error(`No block has been placed : the block is still ${oldBlock.name}`)
+    } else {
+      bot.emit('blockPlaced', [oldBlock, newBlock])
```

Comment:
You don't need to put the parameters between brackets, unless you want them to be a single array parameter.

Reply:
Ok